gtk2_ardour/time_axis_view.h

```diff
@@ -240,6 +240,7 @@ class TimeAxisView : public virtual AxisView
 	std::string            controls_base_unselected_name;
 	std::string            controls_base_selected_name;
 	Gtk::Menu*             display_menu; /* The standard LHS Track control popup-menus */
+	Gtk::MenuItem*         rename_menu_item;
```

Comment:
This is analog to mixer_strip.h. It helps if someone needs to get a handle on rename_menu_item from a less local focus.